Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs

```diff
@@ -45,11 +64,9 @@ public static IServiceCollection ConfigureAppServices(this IServiceCollection se
                     ValidateAudience = false
```

Comment:
What is our Admin validation policy will be? I know that some people use "aud" claim for it. They put audience "user" or "admin" and use it as a role. Others, for example, use custom claim or rely solely on a scope's claim.